web/src/app/app.component.ts

```diff
@@ -25,6 +25,7 @@ export class AppComponent implements OnInit {
       .subscribe((user) => {
         if (user) {
           const userSocial = this.userService.addRefID(user);
+          console.log('credential', user);
```

Comment:
Is this for debugging? Will it be removed now or in the future? If in the future please add a `@TODO` so it doesn't get missed

Reply:
okay Removed @eddiejaoude